### PR TITLE
Feat/port 74.5.36

### PR DESCRIPTION
**Pull request by pabloarodas**
* fix: UAA not respecting identity-zone specific CORS policy #179727364
   * fix: Failing integration tests because of now very restrictive per-zone CORS policy
   * feature: Add option to enforce system zone policy in all zones to CorsFilter
* feat: protect Current-User cookie #179533050
* feature: SameSite=LAX for X-Uaa-Csrf cookie #179533097
* feature: Add Content-Security-Policy header #179533204
   * feature: Move inline javascript to an external .js file

### CODE REVIEW

**Review by hsinn0 — APPROVED** [review]

**Comment by hsinn0** [comment]
Approved.  However, I suggest that we wait for another reviewer's approval before merging.

**Review by peterhaochen47 — APPROVED** [review]
LGTM, just reminder to port the corresponding changes in uaa-release.

**Pull request merged**
